tests/test_config_utils.py

```diff
@@ -36,6 +44,11 @@ def test_bigchain_instance_raises_when_not_configured(monkeypatch):
     # from existing configurations
     monkeypatch.setattr(config_utils, 'autoconfigure', lambda: 0)
 
+    # set the current backend so that Bigchain can create a connection
+    backend = request.config.getoption('--database-backend')
+    backend_conf = getattr(bigchaindb, '_database_' + backend)
+    bigchaindb.config['database'] = backend_conf
```

Comment:
We use this particular bit on setting up the database config a few times, maybe it's worth adding in a fixture?